app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -1,19 +1,28 @@
 module API
   module V2
     class AccessRequestsController < API::V2::ApplicationController
-      before_action :build_access_request
+      before_action :build_access_request, except: :list
+      before_action :build_list_of_access_requests, except: :approve
 
       def approve
         result = AccessRequestApprovalService.call(@access_request)
 
         render status: 200, json: { result: result }
       end
 
+      def list
```

Comment:
As this is the collection the rails convention dictates it should be `index`

Reply:
Have actioned this. Thanks Dan.